scholia/app/templates/sparql/author/publications.sparql

```diff
@@ -0,0 +1,25 @@
+#defaultView:Table
+SELECT
+  (MIN(?dates) AS ?date)
+  ?work ?workLabel
+  (GROUP_CONCAT(DISTINCT ?type_label; separator=", ") AS ?type)
+  (SAMPLE(?pages_) AS ?pages)
+  ?venue ?venueLabel
+  (GROUP_CONCAT(DISTINCT ?author_label; separator=", ") AS ?authors)
+  (CONCAT("../authors/", GROUP_CONCAT(DISTINCT SUBSTR(STR(?author), 32); separator=",")) AS ?authorsUrl)
+WHERE {
+  ?work wdt:P50 wd:{{ q }} .
+  ?work wdt:P50 ?author .
+  OPTIONAL {
+    ?author rdfs:label ?author_label_ . FILTER (LANG(?author_label_) = 'en')
+  }
+  BIND(COALESCE(?author_label_, SUBSTR(STR(?author), 32)) AS ?author_label)
```

Comment:
@fnielsen @egonw any idea why the code highlighting stops working here? Is there a problem with the SPARQL syntax?

Reply:
I cannot see any reason.

Reply:
Okay. That's not really relevant for the PR, I was just curious